src/audio/base_fw_intel.c

```diff
@@ -351,6 +351,19 @@ static int basefw_set_fw_config(bool first_block,
 	return 0;
 }
 
+static int basefw_set_mic_priv_policy(bool first_block,
+				      bool last_block,
+				      uint32_t data_offset_or_size,
+				      const char *data)
+{
+#if CONFIG_INTEL_ADSP_MIC_PRIVACY
+	tr_info(&basefw_comp_tr, "basefw_set_mic_priv_policy");
```

Comment:
do we really need a logging print from a function, that does exactly nothing? Besides, if you really need this, Zephyr already prints the function name, no need to print it twice